exonum/src/node/consensus.rs

```diff
@@ -617,6 +617,12 @@ impl NodeHandler {
         let mut fork = self.blockchain.fork();
         {
             let mut schema = Schema::new(&mut fork);
+            if schema.transactions().contains(&hash) && !schema.transactions_pool().contains(&hash)
```

Comment:
> `!schema.transactions_pool().contains(&hash)`

I don't think we need it here.

Reply:
Agreed. 
Also I want to note that this part was copy-pasted from `validate_block_transactions` method, so we should think about it too.